.github/workflows/main.yaml

```diff
@@ -46,7 +52,6 @@ jobs:
 
     - uses: actions/upload-artifact@v2
       with:
-        name: package-windows
```

Comment:
I am -1 on this change... Without an expliccit name, all archive are named "artifact".
WIth a custom  name, I can tell which artifact is from which build.

Please revert this. Thanks!

Reply:
No problem, I've been experimenting a bit with artefacts.